solutions/sum_range.py

```diff
@@ -0,0 +1,41 @@
+# Define the sum_range function
```

Comment:
add this  before you start the """
#!/usr/bin/env python3
# -*- coding: utf-8 -*-